tests_metricflow/snapshots/test_derived_metric_rendering.py/SqlQueryPlan/DuckDB/test_cumulative_time_offset_metric_with_time_constraint__plan0_optimized.sql

```diff
@@ -42,4 +39,23 @@ FROM (
   WHERE subq_17.metric_time__day BETWEEN '2019-12-19' AND '2020-01-02'
   GROUP BY
     subq_17.metric_time__day
-) subq_23
+)
+
+, cm_5_cte AS (
+  -- Compute Metrics via Expressions
+  SELECT
+    metric_time__day
+    , every_2_days_bookers_2_days_ago AS every_2_days_bookers_2_days_ago
```

Comment:
if the metric and measure name are the same do we need to try and alias the measure? In this case the alias doesn't do anything.

Reply:
Right now, the rule is that we always have column aliases. We could change this, but it's existing behavior.